### DISCUSSION TITLE
display amount of maximum achievable bonus hits in osu!taiko results screen

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#25088] [Ideas] [answered]
title, it'd be quite helpful to know how many hits are needed to achieve maximum score on an osu!taiko map

here's how it could look like:

![image](https://github.com/ppy/osu/assets/62819481/b6a351aa-89e2-4553-864d-455c206d3fc9)

**Comment by peppy** [comment]
Not just taiko but all rulesets should show this now that bonus is always capped.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/25096
